src/test/ui/closures/2229_closure_analysis/diagnostics/liveness_unintentional_copy copy.rs

```diff
@@ -0,0 +1,34 @@
+#![feature(capture_disjoint_fields)]
```

Comment:
How come there is a space here in the file name: `liveness_unintentional_copy copy.rs`?

Reply:
i think we were speeding through when pairing with aman and trying to figure out what was going on, and the name kind of slipped through. Also aman does not like the names anyway, he'll want us to change it  later